float64 pos1x
float64 pos1y

float64 pos2x
float64 pos2y


